src/app/api/quickbooks/invoice/invoice.service.ts

```diff
@@ -345,23 +406,23 @@ export class InvoiceService extends BaseService {
         'Id' | 'SyncToken' | 'sparse'
       > = {}
 
-      if (existingCustomer.familyName !== client.familyName) {
-        sparseUpdatePayload.FamilyName = client.familyName
+      if (existingCustomer.familyName !== clientCompany.familyName) {
+        sparseUpdatePayload.FamilyName = clientCompany.familyName
```

Comment:
```suggestion
        sparseUpdatePayload.FamilyName = clientCompany.familyName
```
This is basically the same thing but simpler